src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -57,6 +59,7 @@ export interface GraphWrapperProps {
 	subscriber: (callback: UpdateStateCallback) => () => void;
 	onSelectRepository?: (repository: GraphRepository) => void;
 	onColumnChange?: (name: GraphColumnName, settings: GraphColumnConfig) => void;
+	onSwitchToRef?: (ref: GraphRef) => void;
```

Comment:
I think we should reframe this to be `onDoubleClickRef`

Reply:
Modified in commit: `Renamed command type and function as requested in PR`.